test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/ApplicationWithTagHelpersTest.cs

```diff
@@ -19,27 +16,24 @@ public TagHelperTest(ApplicationWithTagHelpersFixture fixture)
 
         public ApplicationTestFixture Fixture { get; }
 
-        public static IEnumerable<object[]> ApplicationWithTagHelpersData
+        public static TheoryData ApplicationWithTagHelpersData
         {
             get
             {
-                var urls = new[]
+                return new TheoryData<string>
                 {
                     "ClassLibraryTagHelper",
                     "LocalTagHelper",
-                    "NuGetPackageTagHelper",
```

Comment:
In the past this never ran (I have a poor understanding of how `Enumerable.Zip` works). When it did start running, I realized that the tag helper produces Guids as part of the output making it difficult to baseline. I think it's fine to remove this scenario.